test/render.test.js

```diff
@@ -0,0 +1,16 @@
+import test from "ava";
+import { render } from "../src/render.js";
+import axios from "axios";
+
+class mockServer {
```

Comment:
I think for now you could run tests of local fixtures and consider data loading as outside of the scope of the core lib (i.e. lib is given string or stream and e.g. cli or something else is responsible for getting it ...)

Why? That means people can use their favourite library for loading data e.g. fetch, axios whatever.